operations/observability/mixins/self-hosted/dashboards/examples/overview.json

```diff
@@ -112,12 +112,12 @@
       "seriesOverrides": [
         {
           "$$hashKey": "object:119",
-          "alias": "REGULAR",
+          "alias": "Regular",
```

Comment:
We might be able to delete this (along with kots references), but, for a separate PR (likely for SID Team, which was the former Delivery Team).